lib/engine/game/g_18_nl/step/special_token.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/special_token'
+
+module Engine
+  module Game
+    module G18NL
+      module Step
+        class SpecialToken < Engine::Step::SpecialToken
+          def special_token
+            raise GameError, 'Cannot place station on hex or tile with existing track' unless tile.path.empty?
```

Comment:
I think this will need to be `tile.paths.empty?`

```suggestion
            raise GameError, 'Cannot place station on hex or tile with existing track' unless tile.paths.empty?
```